acl.c

```diff
@@ -706,6 +706,11 @@ int ci_acl_typelist_init(struct ci_acl_type_list *list)
     return 1;
 }
 
+void ci_acl_typelist_destroy(struct ci_acl_type_list *list)
+{
+    free(list->acl_type_list);
```

Comment:
The object 'list' still is available for use after this function is called. So please set the list->acl_type_list to NULL and also reset the other fields of list object to. It is not required as is the c-icap code however maybe in future causes bugs. 
For example:
```
list->acl_type_list = NULL;
list->acl_type_list_size = 0;
list->acl_type_list_num = 0; 
```